src/gui/properties/propertieswidget.cpp

```diff
@@ -830,52 +833,48 @@ void PropertiesWidget::deleteSelectedUrlSeeds()
 
 void PropertiesWidget::copySelectedWebSeedsToClipboard() const
 {
-    const QList<QListWidgetItem *> selected_items = m_ui->listWebSeeds->selectedItems();
-    if (selected_items.isEmpty())
-        return;
+    const QList<QListWidgetItem *> selectedItems = m_ui->listWebSeeds->selectedItems();
+    if (selectedItems.isEmpty()) return;
 
-    QStringList urls_to_copy;
-    foreach (QListWidgetItem *item, selected_items)
-        urls_to_copy << item->text();
+    QStringList URLsToCopy;
```

Comment:
I guess its just me uncomfortable with uppercase variable names.

Reply:
There is project coding style rule about it. Var names should be camelCased (starting with lower case letter).

Reply:
Sure i'll change it but i'm pretty sure glassez did it in RSS.

Reply:
@thalieht, I prefer the well formed abbreviation, if it does not break the coding style (e.g. getURL instead of getUrl). In case I broke coding style in some place we can fix it as well.